packages/ai/README.md

```diff
@@ -1,21 +1,233 @@
 # @rsdoctor/mcp-server
 
-## Usage with Claude Desktop or other MCP clients
+## Introduction
 
-Add to your claude_desktop_config.json or .cursor/mcp.json:
+@rsdoctor/mcp-server is an MCP Server tool designed to help users more conveniently utilize Rsdoctor's build data. It works with Rsdoctor's local build analysis data and helps you quickly obtain build analysis results through a Q&A interface.
+
```

Comment:
We add a link to the Chinese readme, refer to: https://github.com/web-infra-dev/rsbuild/blob/main/README.md?plain=1#L15